script/libtest.sh

```diff
@@ -41,12 +41,21 @@ k8s_wait_for_deployment() {
     
     debug "Waiting for deployment ${deployment} to be successfully rolled out..."
     # Avoid to exit the function if the rollout fails
-    silence kubectl rollout status --namespace "$namespace" deployment "$deployment" || exit_code=$?
+    silence kubectl rollout status --namespace "$namespace" deployment "$deployment" -w --timeout=60s || exit_code=$?
```

Comment:
These changes are not strictly necessary (the main error was I didn't pass the db password :facepalm: ), but I think they make the function work as expected, I mean, performing an actual waiting, rather than digesting the exit_code for logging purposes.